index.html

```diff
@@ -55,6 +54,9 @@
 </nav>
 </header>
 
+<div class="throbber-loader" style="center">
+  <p>Loading…</p>
+</div>
```

Comment:
This is something that only needs to be displayed during an AJAX call. Currently you have it displaying at all times.